src/styles/blocks/card.scss

```diff
@@ -0,0 +1,117 @@
+@import "../utils/extends";
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 1200px;
+
+  &--padding-top--50px {
+    padding-top: 50px;
+  }
+
+  &--margin--auto {
+    margin: auto;
+  }
+
+  &__item {
+    display: flex;
+    flex-wrap: wrap;
+    width: 200px;
+    padding-top: 32px;
+    background: #fff;
+    border: 1px solid #f3f3f3;
```

Comment:
seperate catalog and one independent card as different BEM blocks

Reply:
Done